package/tests/testthat/_snaps/pkpt03/markdown-snaps.md

```diff
@@ -4,37 +4,81 @@
       print(data_snap[[i]])
     Output
       Summary of Plasma PK Parameter by Treatment Arm, PK Population
-      Analyte: Plasma Drug X 
-      Visit: CYCLE 1 DAY 1
+      Analyte: Plasma Drug X
       
-      ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
-      Treatment Arm                                                                                                                             
-        PK Parameter              n      Mean         SD       CV (%)   Geometric Mean   CV % Geometric Mean    Median      Minimum     Maximum 
-      ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
-      ARM A                                                                                                                                     
-        Cmax (ug/mL)             134   3.025e+01   6.239e+00    20.6      2.961e+01             21.0           2.986e+01   1.753e+01   4.871e+01
-        AUCinf obs (day*ug/mL)   134   2.028e+02   3.766e+01    18.6      1.994e+02             18.7           1.971e+02   1.253e+02   3.110e+02
-        CL obs (ml/day/kg)       134   5.043e+00   1.041e+00    20.6      4.929e+00             22.4           5.078e+00   2.255e+00   7.395e+00
-      ARM C                                                                                                                                     
-        Cmax (ug/mL)             132   3.004e+01   5.457e+00    18.2      2.954e+01             18.9           2.977e+01   1.585e+01   4.757e+01
-        AUCinf obs (day*ug/mL)   132   1.955e+02   3.785e+01    19.4      1.917e+02             20.1           1.962e+02   1.030e+02   3.145e+02
-        CL obs (ml/day/kg)       132   5.009e+00   9.846e-01    19.7      4.907e+00             21.1           4.969e+00   2.102e+00   7.489e+00
+      ————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+      Treatment Arm                                                                                                               
+        PK Parameter                n    Mean    SD     CV (%)   Geometric Mean   CV % Geometric Mean   Median   Minimum   Maximum
```

Comment:
Hi @anajens, I think the top left should be VISIT.

Reply:
Good catch @qiliu1013! I changed AVISIT to be a page-by variable following the example with real data from https://github.com/insightsengineering/sme-tasks/issues/641 but forgot to update the `result` variable. When you have a  page-by variable and print the non-paginated table the page-by label was not shown. 

Does the layout look OK now?

Reply:
@anajens, can we you please send me a creenshot of the situation you mentioned?

Reply:
Resolved offline.